Semesters scroll on overflow

adeniji19 opened the pull request.

adeniji19 committed:
Semesters scroll on overflow

churichard reviewed (changes requested):
Making the window width smaller introduces blank space on the right side of the schedule - it's very egregious when the window size is small:
<img width="751" alt="image 2018-05-07 at 12 12 30 am" src="https://user-images.githubusercontent.com/4218237/39684649-6269a59a-518b-11e8-8b21-26f9588cb1a5.png">

churichard committed:
Add popover with course info, add grabbing cursor (#33)

bnehoran committed:
Add support for double-counting courses (#35)

* Fixed only one course being added to each dist_req

* Added support for double counting

* Fixed AB language requirement by changing to a course_list.

* Fixed named vs unnamed/hidden BSE distribution requirements.

* refactored code to pass in req to markers and check num_settlable

* No longer adding reqs of double-counted courses to course["settled"]

* Corrected typo in num_settleable

churichard committed:
Fix typo in BSE json and change wording in verifier degree progress (#38)

bnehoran committed:
Fix auto-settling of distribution requirements (#39)

* Fix typo in BSE json and change wording in verifier degree progress (#38)

* Allow semester lists shorter than 8 semesters.

* Add testing code for AB/BSE requirements.

* add a test testing BSE requirements on a COS student

* fixed duplicate courses only appearing once in unsettled lists

* Fixed distribution requirements not auto-settling.

* Add a test for distribution requirements auto-settling.

bnehoran committed:
Fix diff fallback in tester.py (#40)

bnehoran committed:
Rename majors and certificates directory (#42)

* Rename the majors-and-certificates directory to majors_and_certificates

* Fix relative local import in verifier.py

danielleung27 committed:
Added requirements pane

Implemented requirements tree display, added course settling functionality, changed color coding.

adeniji19 committed:
scoll responsive to screen size

adeniji19 committed:
better scroll

churichard reviewed (changes requested):
Not sure if this is something that you did, but popovers seem to be a bit messed up now (this is hovering over a course in the first semester):
<img width="1440" alt="image 2018-05-08 at 12 58 13 pm" src="https://user-images.githubusercontent.com/4218237/39771269-830bb880-52bf-11e8-9b15-f12e5959aece.png">
